cpp/src/copying/get_element.cu

```diff
@@ -122,7 +124,24 @@ struct get_element_functor {
     rmm::cuda_stream_view stream,
     rmm::mr::device_memory_resource *mr = rmm::mr::get_current_device_resource())
   {
-    CUDF_FAIL("get_element_functor not supported for list_view");
+    rmm::device_scalar<bool> d_valid;
+    auto d_input = column_device_view::create(input);
+    device_single_thread([d_input = *d_input, index, valid = d_valid.data()] __device__() {
+      *valid = d_input.is_valid(index);
+    });
+    bool valid = d_valid.value(stream);
```

Comment:
I feel like this code should be in a utility function... I think reading back the 32-bit bitmask word that includes this element to the host and then masking off the right bit would be faster than allocating a device_scalar, launching a kernel, and then copying the result back and syncing.

Reply:
There's `bit_is_set` in `utilities/bin.hpp`.